cadc-vos-server/src/main/java/org/opencadc/vospace/server/PathResolver.java

```diff
@@ -105,9 +105,8 @@ public class PathResolver {
     private final VOSpaceAuthorizer voSpaceAuthorizer;
     private final boolean resolveLinks;
     
-    private List<String> visitedPaths = new ArrayList<>();
     private int visitLimit = 20;
-    private int visitCount = 0;
+
 
 
```

Comment:
The class javadoc should not mention "resolving" and there really needs to be a javadoc here for the constructor to explain exactly how `resolveLinks` effects `getNode(...)` -- for our own benefit!

I would change the 3rd arg to `resolvelink` (singular since it only effects the last name in the target path).

Optional: since most of the code (actions) use resolveLink = false, it might be a good idea to have these ctors instead:
```
PathResolver(NodePersistence nodePersistence, VOSpaceAuthorizer voSpaceAuthorizer) {
   this(nodePersistence, voSpaceAuthorizer, false);
}

PathResolver(NodePersistence nodePersistence, VOSpaceAuthorizer voSpaceAuthorizer, boolean resolveLink) ...
```
use the shorter one in all the actions and the special case where explicitly needed in transfer code... but maybe that makes things more obscure?

Reply:
Rename the argument and added docs.